### ISSUE TITLE
Errror starting docker-compose file

### ISSUE THREAD

**Issue by leon1995** [pi-hole/docker-pi-hole#521]
This is a... 

- [ ] Request for a new or modified feature
- [x] Issue trying to run the docker image
- [ ] Issue trying to build / test / develop the docker image

## Description

When trying to start the docker-compose file i get an issue
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.pihole.environment contains {"TZ": "Europe/Berlin"}, which is an invalid type, it should be a string
```

## Debug steps I have tried

replacing ' ' with " "

my `docker-compose.yml` file looks like this
```
version: "3"

# More info at https://github.com/pi-hole/docker-pi-hole/ and https://docs.pi-hole.net/
services:
  pihole:
    container_name: pihole
    image: pihole/pihole:latest
    volumes:
      - './etc-pihole/:/etc/pihole/'
      - './etc-dnsmasq.d/:/etc/dnsmasq.d/'
    environment:
      - TZ: 'Europe/Berlin'
      - WEBPASSWORD: "testtest"
      - VIRTUAL_HOST=pihole.mydomain.de
      - VIRTUAL_NETWORK=nginx-proxy
      - VIRTUAL_PORT=3232
      - LETSENCRYPT_HOST=pihole.mydomain.de
      - LETSENCRYPT_EMAIL=mail@mydomain.de
    dns:
      - 127.0.0.1
      - 1.1.1.1
    networks:
      - proxy-tier
    restart: always

networks:
  proxy-tier:
    external:
      name: nginx-proxy
```

**Comment by STaRDoGG** [comment]
For kicks, see what happens if you wrap the values of anything with a "." or a "-", or "@" in it with double quotes.

i.e. 
```
      - VIRTUAL_HOST="pihole.mydomain.de"
      - VIRTUAL_NETWORK="nginx-proxy"
      - VIRTUAL_PORT=3232
      - LETSENCRYPT_HOST="pihole.mydomain.de"
      - LETSENCRYPT_EMAIL="mail@mydomain.de"
```

**Comment by diginc** [comment]
TZ and WEBPASSWORD shouldn't have a colon+space `: ` after them.  Change the colon+space to just an equal sign like the other 5 correct env vars.

**Comment by STaRDoGG** [comment]
> TZ and WEBPASSWORD shouldn't have a colon+space `: ` after them. Change the colon+space to just an equal sign like the other 5 correct env vars.

I'm not sure that's the problem (though it's still worth a shot). 

It's got a colon + space in the example on it's [Docker Hub page](https://hub.docker.com/r/pihole/pihole), and I copied that verbatim, then slightly tweaked it to my own preferences, but left those 2 with the colon + space, and I'm up and running without that error. I did wrap my password in double quotes though, so that's worth a shot.
